packages/core/src/forms/control-label/control-label.element.scss

```diff
@@ -33,7 +33,6 @@
   font-size: $cds-global-typography-secondary-font-size !important;
   font-weight: $cds-global-typography-secondary-font-weight !important;
   color: $cds-global-typography-color-300 !important;
-  line-height: $cds-global-typography-secondary-line-height !important;
```

Comment:
What happens if the label wraps? Does everything still look okay with the label with whatever line-height it's inheriting?

Reply:
We will need to keep a line height so it doesn't inherit from the host document

Reply:
Thanks, I needed to bump it to --cds-global-sapce-2 to align with the label text. Image below.